lib/improver/ensemble_copula_coupling/ensemble_copula_coupling.py

```diff
@@ -134,7 +136,7 @@ def _add_bounds_to_percentiles_and_forecast_at_percentiles(
         """
         lower_bound, upper_bound = bounds_pairing
         percentiles = insert_lower_and_upper_endpoint_to_1d_array(
-            percentiles, 0, 1)
+            percentiles, 0, 100)
```

Comment:
I think I've now corrected this behaviour, so that all percentiles written into cubes are in percentage format. I do temporarily convert percentiles to be between 0 and 1 for interpolation reasons, although this should be reviewed when the CLI for the probability plugins is implemented.